traceapp/vis.go

```diff
@@ -92,6 +92,11 @@ func (a *App) d3timelineInner(t *appdash.Trace, depth int) ([]timelineItem, erro
 	}
 	for _, e := range events {
 		if e, ok := e.(appdash.TimespanEvent); ok {
+			// Continue to next iteration
+			// if e.Start() or e.End() are empty time values.
+			if e.Start() == (time.Time{}) || e.End() == (time.Time{}) {
+				continue
+			}
```

Comment:
> Just for the record, why is that it would be better to still show the timespan even there's no start or end for it?

The motivation is that the span might still contain valuable information (log messages, etc in `Verbose Data View` under web UI).

I guess it's hard to say which way is better, though. How about we settle for just adding a `log.Println` statement saying `Found a TimespanEvent with invalid/zero times.` ? (that way we still know something went wrong).